.changeset/cyan-guests-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@skeletonlabs/floating-ui-svelte": patch
```

Comment:
I'd suggest we follow the same pattern as Skeleton and not add Changesets to documentation pages. Only for the library. The PR templates are setup per this recommendation.